iq/questions/tests.py

```diff
@@ -63,20 +50,40 @@ def test_valid_question_form(self):
             "difficulty": "5",
             "tags": "asdf,fdsa,asdfasdf"
         })
-        form.is_valid()
+        self.assertTrue(form.is_valid())
```

Comment:
That's a good assert to have. It will save you some debugging time if a later assert fails.